Client/Frontend/Settings/Clearables.swift

```diff
@@ -42,27 +42,38 @@ class HistoryClearable: Clearable {
 
         // Treat desktop sites as part of browsing history.
         Tab.ChangeUserAgent.clear()
+        switch profile.historyApiConfiguration {
+        case .old:
+            return profile.history.clearHistory().bindQueue(.main) { success in
+                return self.clearAfterHistory(success: success)
+        }
+        case.new:
+                _ = profile.history.clearHistory()
+                return profile.places.deleteEverythingHistory().bindQueue(.main) { success in
+                    return self.clearAfterHistory(success: success)
```

Comment:
indentation and space with `case.new:` I see some other places where indentation is off but to avoid being repetitive I wont put a comment. I normally use `ctrl+i` shortcut for this to make it easy and quick (select code then auto indent).
```suggestion
            }
        case .new:
            _ = profile.history.clearHistory()
            return profile.places.deleteEverythingHistory().bindQueue(.main) { success in
                return self.clearAfterHistory(success: success)
            }
```

Reply:
WOW the `ctrl+i` shortcut was very handy!! thank you 🙏